pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java

```diff
@@ -568,6 +568,8 @@ public static class Server {
     public static final String CONFIG_OF_SWAGGER_USE_HTTPS = "pinot.server.swagger.use.https";
     public static final String CONFIG_OF_ADMIN_API_PORT = "pinot.server.adminapi.port";
     public static final int DEFAULT_ADMIN_API_PORT = 8097;
+    public static final String CONFIG_OF_ADMIN_API_RESOURCE_PACKAGES = "pinot.server.adminapi.resource.packages";
```

Comment:
Keeping it consistent with controller and broker
```suggestion
    public static final String CONFIG_OF_SERVER_RESOURCE_PACKAGES = "pinot.server.restlet.api.resource.packages";
```

Reply:
Looks like there is no `pinot` prefix either, 

```
pinot-controller/src/main/java/org/apache/pinot/controller/ControllerConf.java:  public static final String CONTROLLER_RESOURCE_PACKAGES = "controller.restlet.api.resource.packages";
pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java:    public static final String BROKER_RESOURCE_PACKAGES = "broker.restlet.api.resource.packages";
```